### DISCUSSION TITLE
Limit output of text in a widget

### DISCUSSION THREAD

**Discussion by AtiDev17** [TomSchimansky/CustomTkinter#2635] [Q&A] [answered]
Hi,

I wrote a little custom widget which creates a checkbox and button
```python
class AssetWidget(customtkinter.CTkFrame):
    def __init__(self, parent, name = ""):
        super().__init__(parent)

        self.checkbox = customtkinter.CTkCheckBox(self, text="TestArchive")
        self.checkbox.grid(row=0, column=0, padx=20, pady=10, sticky="w")

        self.install_button = customtkinter.CTkButton(self, text="Install")
        self.install_button.grid(row=0, column=1, padx=20, pady=10, sticky="e")

        self.columnconfigure(0, weight=1)
        self.columnconfigure(1, weight=0)
```

I wanted to ask if there is a built-in way to limit the output of a text like this:

![grafik](https://github.com/user-attachments/assets/83ceb9d0-624e-4555-85b4-9ad077c92e02)

Right now if the text is too long it looks like this with my piece of code:

![grafik](https://github.com/user-attachments/assets/16ea2d4c-5fa3-4d6c-b6a8-3b446df028f8)

I know that I could create my own method to replace the output string before it gets rendered but I wanted to know if this or similar functionality is already implemented in customtkinter.

Kind Regards

**Comment by dipeshSam** [comment, marked as answer]
There is no built-in support for ellipsis (`...`) text in `customtkinter`.

Regards.
